tests/workflows/fastq_download_prefetch_fasterqdump_sratools/main.nf.test

```diff
@@ -32,6 +32,42 @@ nextflow_workflow {
                 { assert workflow.success },
                 { assert snapshot(workflow.out).match() }
             )
+
+            assert path("$launchDir/output").exists()
+
+            def lines = path("$launchDir/output/prefetch/failures_report.csv").readLines()
```

Comment:
If there's meant to be no errors for this test case I think you could instead assert the file is empty (or only contains the header line)?

Reply:
Updated in https://github.com/phac-nml/fetchdatairidanext/commit/bbf2e3fe013b902b060af3ca063f5fad25f837fd

(checks for specific number of lines)